kivy/core/audio/audio_android.py

```diff
@@ -0,0 +1,70 @@
+"""
+AudioAndroid: Kivy audio implementation for Android using native API
+"""
+
+__all__ = ("SoundAndroidPlayer", )
+
+from jnius import autoclass
+from kivy.core.audio import Sound, SoundLoader
+
+
+MediaPlayer = autoclass("android.media.MediaPlayer")
+AudioManager = autoclass("android.media.AudioManager")
+
+
+class SoundAndroidPlayer(Sound):
+    @staticmethod
+    def extensions():
+        return ("mp3", "mp4", "aac", "3gp", "flac", "mkv", "wav", "ogg", "m4a",
+                "gsm", "mid", "xmf", "mxmf", "rtttl", "rtx", "ota", "imy")
+
+    def __init__(self, **kwargs):
+        self._mediaplayer = None
+        super(SoundAndroidPlayer, self).__init__(**kwargs)
+
+    def load(self):
+        self.unload()
+        self._mediaplayer = MediaPlayer()
+        self._mediaplayer.setAudioStreamType(AudioManager.STREAM_MUSIC);
```

Comment:
One more thing: setAudioStreamType was deprecated in API 26. I suggest you to use [setAudioAttributes](https://developer.android.com/reference/android/media/MediaPlayer.html#setAudioAttributes(android.media.AudioAttributes)) for API >= 26 and setAudioStreamType for backwards compatibility.

Reply:
Do you have a conversation example that would work for us?

Reply:
```suggestion
        if api_version >= 21:
            AudioAttributesBuilder = autoclass(
                "android.media.AudioAttributes$Builder"
            )
            self._mediaplayer.setAudioAttributes(
                AudioAttributesBuilder()
                .setLegacyStreamType(AudioManager.STREAM_MUSIC)
                .build()
            )
        else:
            self._mediaplayer.setAudioStreamType(AudioManager.STREAM_MUSIC)

```

Reply:
I put AudioAttributesBuilder from API >=21 instead of >=26 because it's when it was first added.